<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_StockCloths">
         <property name="font">
          <font>
           <family>Adobe Clean</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: rgb(209, 238, 252);
  color: black; 
  border-right: 0;
  border-left: 0;
  border-top: 0;
  border-bottom: 0;
  height: 24;
}

QPushButton:pressed {
    background-color: rgb(165, 190, 202);
}

QPushButton:hover:!pressed
{
	 background-color: rgb(187, 214, 228);
}</string>
         </property>
         <property name="text">
          <string>Склад Одежды</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_StockFurniture">
         <property name="font">
          <font>
           <family>Adobe Clean</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: rgb(209, 238, 252);
  color: black; 
  border: 0;
  height: 24;
}

QPushButton:pressed {
    background-color: rgb(165, 190, 202);
}

QPushButton:hover:!pressed
{
	 background-color: rgb(187, 214, 228);
}</string>
         </property>
         <property name="text">
          <string>Склад Фурнитуры</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_StockProducts">
         <property name="font">
          <font>
           <family>Adobe Clean</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: rgb(209, 238, 252);
  color: black; 
  border-right: 0;
  border-left: 0;
  border-top: 0;
  border-bottom: 0;
  height: 24;
}

QPushButton:pressed {
    background-color: rgb(165, 190, 202);
}

QPushButton:hover:!pressed
{
	 background-color: rgb(187, 214, 228);
}</string>
         </property>
         <property name="text">
          <string>Склад Изделий</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox{
  border: 1px solid #A5BECA;
  /*border-radius: 6px;*/
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="horizontalWidget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
qproperty-alignment: AlignCenter;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QListWidget" name="listWidget_Products">
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QListWidget" name="listWidget_Furniture">
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget_Cloths">
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
